lib/constants/branch-constants.ts

```diff
@@ -1,8 +1,5 @@
-// Branch Status
-export const BRANCH_STATUS_SUCCESS = 'success';
-export const BRANCH_STATUS_FAILURE = 'failure';
-export const BRANCH_STATUS_ERROR = 'error';
-export const BRANCH_STATUS_PENDING = 'pending';
-export const BRANCH_STATUS_CREATED = 'created';
-export const BRANCH_STATUS_RUNNING = 'running';
-export const BRANCH_STATUS_FAILED = 'failed';
+export enum BranchStatus {
+  green = 'green', // 'success'
+  yellow = 'yellow', // 'created', 'running'
+  red = 'red', //  'error', 'failed'
+}
```

Comment:
Moving to `lib/types` instead of `lib/constants` cause it not really a constant? 🤔

Reply:
Sure.